wagtailio/static/sass/components/_headline.scss

```diff
@@ -73,13 +77,24 @@
         position: absolute;
         width: 230px;
         height: 300px;
-        top: 0;
-        right: -40px;
+        right: -20px;
```

Comment:
Nitpick: It would be nice to keep consistent to either `% or px`, that's probably me just being neurotic but I feel like it's easier to understand the difference between the mobile and the desktop position.

Reply:
I'm happy to keep them as `px` - feels more predictable to me but happy either way.